fetch/fetch.go

```
@@ -365,3 +407,34 @@ func initStatusEntry(
 
 	return fetchStatus, nil
 }
+
+// TODO: handle the case where the file override happens.
```

File override should probably be handled inside the fetch table stuff. The fileName parameter is is the thing we use to know what to start at.

---

Discussed this offline with @Jeremyyang920 . Plan here is two fold:
1. Do a run pre-check validation so that if an invalid continuation file name is passed in, we fail early.
2. In the helper here, update the exceptionLog file with the passed in cfg.ContinuationFileName if it is set.

---

This is what the error message looks like:
```
continuation file name "asdk" doesn't match the file convention "part_\d+(\.csv|\.tar\.gz)"
exit status 1
zsh: command not found: lf
(⎈|rancher-desktop:N/A)ryanluu@crlMBP-XFFK6Q9QL6MTk1 molt % 
```

---

I think this will be something we have to document for this feature for sure in the readme or official docs of file naming

---

Yep makes sense to me! The error message indicates this case already